kuma/users/jinja2/users/includes/user_title_info.html

```diff
@@ -70,4 +70,38 @@ <h1 class="user-title">
   </ul>
 </div>
 {% endif %}
-<div class="user-since">{% trans date_joined=datetimeformat(detail_user.date_joined, format='date') %}Member since {{ date_joined }}{% endtrans %}</div>
\ No newline at end of file
+
+<ul class="user-links">
+  {% if detail_user.twitter_url %}
+    <li class="twitter"><i class="icon-twitter" aria-hidden="true"></i><a href="{{ detail_user.twitter_url }}" rel="me external" class="url">{{ _("Twitter") }}</a></li>
+  {% endif %}
+
+  {% if accounts.github %}
+    {% for github_account in accounts.github %}{% if github_account.extra_data %}
+    <li class="github"><i class="icon-github" aria-hidden="true"></i><a href="{{ github_account.get_profile_url() }}" rel="me external" class="url">{{ _("GitHub") }}</a></li>
```

Comment:
The `aria-hidden` attributes are not being updated when these items are visible.